src/domains/my/detail/components/MyDraftsList.jsx

```
@@ -1,19 +1,74 @@
+        day: '2-digit',
+      })
+      .split('. ')
+      .join('-')
+      .slice(0, -1) // '2025. 06. 02.' -> '2025-06-02'
+  } catch (error) {
+    console.error('날짜 포맷팅 오류:', error)
+    return '날짜 오류'
+  }
+}
+
+const MyDraftsList = ({ draftsListData, onDraftClick }) => {
```

ESLint

[error] 'draftsListData' is missing in props validation

(react/prop-types)